pkgs/servers/overseerr/default.nix

```diff
@@ -0,0 +1,80 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+, nodejs
+, python3
+, makeWrapper
+}:
+
+mkYarnPackage rec {
+  pname = "overseerr";
+  version = "1.33.2";
+
+  src = fetchFromGitHub {
+    owner = "sct";
+    repo = "overseerr";
+    rev = "v${version}";
+    hash = "sha256-xDzWyU4f56+0Tpk87LpH6zXtxmRxVMCKySCY6WD5go0=";
+  };
+
+
+  offlineCache = fetchYarnDeps {
+    yarnLock = "${src}/yarn.lock";
+    hash = "sha256-SZwhC6djgU5qshtDhQnkz/INeklp/c+BKjn7ao0r5IE=";
+  };
+
+  env = {
+    CYPRESS_INSTALL_BINARY = 0; # cypress tries to download binaries otherwise
+  };
+
+  nativeBuildInputs = [ nodejs makeWrapper ];
+
+  # Fixes "SQLite package has not been found installed" at launch
+  pkgConfig.sqlite3 = {
+    nativeBuildInputs = [ nodejs.pkgs.node-pre-gyp python3 ];
+    postInstall = ''
+      export CPPFLAGS="-I${nodejs}/include/node"
+      node-pre-gyp install --prefer-offline --build-from-source --nodedir=${nodejs}/include/node
+      rm -r build-tmp-napi-v6
+    '';
+  };
+
+  # Fixes MODULE_NOT_FOUND at launch.
+  pkgConfig.bcrypt = {
+    nativeBuildInputs = [ nodejs.pkgs.node-pre-gyp python3 ];
+    postInstall = ''
+      export CPPFLAGS="-I${nodejs}/include/node"
+      node-pre-gyp install --prefer-offline --build-from-source --nodedir=${nodejs}/include/node
+    '';
+  };
+
+  postInstall = ''
+    makeWrapper '${nodejs}/bin/node' "$out/bin/overseerr" \
+      --add-flags "$out/libexec/overseerr/deps/overseerr/dist/index.js" \
+      --set NODE_ENV production
+  '';
+
+  buildPhase = ''
+    runHook preBuild
+    (
```

Comment:
pushd and popd are better suited

Reply:
Wouldn't you have to manually disable `dotglob` as well then, in order no to infect `postBuild`?

Reply:
Why it would be problematic?

Reply:
I'm not sure whether you're asking if manually disabling `dotglob` would be problematic, or whether leaving it on would.

I don't think manually disabling it is a problem, but opening a subshell seems to me like a cleaner solution since it guarantees any state changes gets reverted at the end.

As for whether leaving `dotglob` on is a problem, I don't think anyone would ever realistically be affected by this one instance. But being able to assume default shell settings when you override any phase for any package is a nice quality to provide.